osu.Game/Configuration/OsuConfigManager.cs

```diff
@@ -211,7 +211,7 @@ protected override void InitialiseDefaults()
             SetDefault(OsuSetting.LastProcessedMetadataId, -1);
 
             SetDefault(OsuSetting.ComboColourNormalisationAmount, 0.2f, 0f, 1f, 0.01f);
-            SetDefault<UserStatus?>(OsuSetting.UserOnlineStatus, null);
+            SetDefault(OsuSetting.UserOnlineStatus, UserStatus.Online);
```

Comment:
Note that this will throw a non-fatal runtime error if you have the previous `null` default value in the config:
```
UserOnlineStatus =
```
However, the value is set to `Online` as soon as the user logs in.

This is not easy to handle because we don't have a good way to change values from nullable to non-nullable right now (if ever?), so I left it as is and hope this is okay. Adding a migration doesn't help because those are processed after all these defaults are applied.